package.json

```diff
@@ -91,7 +91,7 @@
   },
   "homepage": "https://renovatebot.com",
   "engines": {
-    "node": "^10.13.0 || ^12.0.0",
+    "node": "^12.0.0 || ^13.0.0",
```

Comment:
```suggestion
    "node": "^12.13.0 || ^14.0.0",
```

Reply:
I'll make the first change, to target the LTS release of v12, but we need to keep v13 in the allowed engines. For a lot of package managers `node` will soon point to v13, which will prevent running renovate unless you add the `--ignore-engines` flag.

Reply:
Ok, but we should at least raise to 13.3.0 as a minimum

Reply:
Yep, your edited suggestion looks good.

Reply:
I don't think this is resolved though? Should it be `"node": "^12.13.0 || ^13.3.0",` ?

Reply:
No, was a typo. Min version should 12.3.0 vor lts and 13.0.0 vor next